src/main/java/com/minecolonies/coremod/client/gui/WindowHireWorker.java

```diff
@@ -9,10 +9,12 @@
 import com.minecolonies.api.colony.IColonyView;
 import com.minecolonies.api.colony.buildings.HiringMode;
 import com.minecolonies.api.entity.citizen.Skill;
+import com.minecolonies.api.util.constant.BuildingConstants;
 import com.minecolonies.api.util.constant.Constants;
 import com.minecolonies.coremod.Network;
 import com.minecolonies.coremod.colony.CitizenDataView;
 import com.minecolonies.coremod.colony.buildings.AbstractBuildingWorker;
+import com.minecolonies.coremod.colony.buildings.workerbuildings.BuildingArchery;
```

Comment:
Watch out for autoinserted import statements.  If you're using IntelliJ, I try to get in the habit of checking the Problems tab (alt-6 by default) for each changed file, since otherwise they'll end up collapsed out of view.  There's a lot of false positives there (it's _way_ too aggressive on typos, most obviously), but a quick way to check a bunch of files without having to scan them by eyeball.

Reply:
Extraneous imports removed in new commit.